### PR TITLE
Move CT incidence from net income to tax

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
This ensures it appears in the waterfall chart on policyengine.

### CODE REVIEW

File: openfisca_uk/variables/finance/income.py [hunk 1]
```diff
@@ -102,10 +102,7 @@ class net_income(Variable):
 
     def formula(person, period, parameters):
         income = person("gross_income", period) - person("tax", period)
```
Comment (MaxGhenis) [inline openfisca_uk/variables/finance/income.py:104]:
```suggestion
        return person("gross_income", period) - person("tax", period)
```

File: openfisca_uk/variables/finance/tax/tax.py [hunk 2]
```diff
@@ -9,8 +9,16 @@ class tax(Variable):
     definition_period = YEAR
 
     def formula(person, period, parameters):
-        return person("income_tax", period) + person(
-            "national_insurance", period
+        ct_payment = person("is_household_head", period) * person.household(
+            "council_tax", period
+        )
+        return (
+            add(
```
Comment (MaxGhenis) [inline openfisca_uk/variables/finance/tax/tax.py:16]: Could be more readable as a separate `income_tax_ni` variable

**Review by MaxGhenis — APPROVED** [review]
nits

File: openfisca_uk/variables/finance/tax/tax.py [hunk 3]
```diff
@@ -9,19 +9,25 @@ class tax(Variable):
...
+class household_tax(Variable):
+    value_type = float
+    entity = Household
+    label = u"Total tax"
+    definition_period = YEAR
+    unit = "currency-GBP"
+
+    def formula(household, period, parameters):
+        personal_taxes = aggr(household, period, ["tax"])
+        return max_(0, personal_taxes + household("council_tax", period))
```
Comment (MaxGhenis) [inline openfisca_uk/variables/finance/tax/tax.py:28]: Why `max_(0, ...)`?

File: openfisca_uk/variables/finance/tax/tax.py (hunk 2 above)
Comment (nikhilwoodruff) [reply openfisca_uk/variables/finance/tax/tax.py]: Yeah - this would involve a lot of changes/renames as `tax` is used quite a lot throughout.

Comment (MaxGhenis) [reply openfisca_uk/variables/finance/tax/tax.py]: just mean python variable not openfisca variable

File: openfisca_uk/variables/finance/tax/tax.py (hunk 3 above)
Comment (nikhilwoodruff) [reply openfisca_uk/variables/finance/tax/tax.py:28]: Oh actually probably unnecessary

**Pull request merged**